packages/cli/src/commands/governance/propose.test.ts

```diff
@@ -174,7 +174,7 @@ testWithGanache('governance:propose cmd', (web3: Web3) => {
         '--jsonTransactions',
         'transactions.json',
         '--deposit',
-        '1000000000000000000',
+        '1000000000000000000', // TODO(Arthur): This is 1 CELO, shouldn't this be 10,000 CELO now?
```

Comment:
the devchain we are on can have different params than is on alfajores or mainnet

Reply:
As you said, it looks like the `minDeposit` on ganache is current 1

https://github.com/celo-org/developer-tooling/blob/ec6191045e1ad671fa87707b2a654cd47b191bfa/packages/dev-utils/src/migration-override.json#L25-L25

Reply:
Removed the comment in https://github.com/celo-org/developer-tooling/pull/233/commits/c1e8be75125cba6a9a6c8e94dafc88b70aa34f53, and left the value unchanged.